app/src/local-resources/runs/constants.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  RUN_STATUS_AWAITING_RECOVERY,
+  RUN_STATUS_AWAITING_RECOVERY_BLOCKED_BY_OPEN_DOOR,
+  RUN_STATUS_AWAITING_RECOVERY_PAUSED,
+  RUN_STATUS_BLOCKED_BY_OPEN_DOOR,
+  RUN_STATUS_FAILED,
+  RUN_STATUS_FINISHING,
+  RUN_STATUS_IDLE,
+  RUN_STATUS_PAUSED,
+  RUN_STATUS_RUNNING,
+  RUN_STATUS_STOP_REQUESTED,
+  RUN_STATUS_STOPPED,
+  RUN_STATUS_SUCCEEDED,
+} from '@opentrons/api-client'
+
+import type { RunStatus } from '@opentrons/api-client'
+
+export const START_RUN_STATUSES: RunStatus[] = [
+  RUN_STATUS_IDLE,
+  RUN_STATUS_PAUSED,
+  RUN_STATUS_BLOCKED_BY_OPEN_DOOR,
+]
+export const RUN_NOT_STARTED_STATUSES: RunStatus[] = [
+  RUN_STATUS_IDLE,
+  RUN_STATUS_BLOCKED_BY_OPEN_DOOR,
+]
+export const NEEDS_CONFIRMATION_STATUSES: RunStatus[] = [
```

Comment:
```suggestion
export const RUN_MODULE_REQUIRES_CONFIRM_STATUSES: RunStatus[] = [
```
Nit, I think if we name this group of statuses as such, we'll be able to more broadly apply and generalize this util/constant group in the future.